content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,255 @@
+---
+title: 'Get Started: Experiments'
+---
+
+# Get Started with Experiments
+
+In machine learning projects, number of <abbr>experiments</abbr> grow rapidly.
+DVC can track these experiments, list their most relevant parameters and
+metrics, and commit only the ones that we need to Git.
+
+In this section, we explore the basic features of DVC experiment management with
+[`get-started-experiments`][gse] project.
+
+[gse]: https://github.com/iterative/get-started-experiments
+
+<details>
+
+## ⚙️ Installing the Example Project
+
+These commands are run in the [`get-started-experiments`][gse] project. You can
+run the commands in this document after cloning the repository and installing
+the requirements.
+
+### ⚡ Clone the project and create virtual environment
+
+Please clone the project and create a virtual environment.
+
+> We create a virtual environment to keep the libraries we use isolated from the
+> rest of your system. This prevents version conflicts.
+
+```dvc
+$ git clone https://github.com/iterative/get-started-experiments -b get-started
+$ cd get-started-experiments
+$ virtualenv .venv
+$ . .venv/bin/activate
+$ python -m pip install -r requirements.txt
+```
+
+### ✅ Get the data set
+
+The repository we cloned doesn't contain the dataset. We use `dvc pull` to
+update the missing data files.
+
+```dvc
+$ dvc pull
+```
+
+The repository already contains the necessary configuration to run the
+experiments.
+
+## 💡 Preparing a project for DVC experiments
```

Comment:
It's not immediately clear to me that these steps are already complete for the cloned repo. I guess this is more of a reference so that users know how you got to this point? I think it's important to clarify, or my first step would be `dvc init`, which will throw an error.